test/Microsoft.SqlTools.ServiceLayer.Test/QueryExecution/Execution/RequestParamTests.cs

```diff
@@ -0,0 +1,78 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+//using System;
+//using Microsoft.SqlTools.ServiceLayer.QueryExecution.Contracts.ExecuteRequests;
+//using Microsoft.SqlTools.ServiceLayer.SqlContext;
+//using Microsoft.SqlTools.ServiceLayer.Workspace;
```

Comment:
Please remove this file since it's entirely commented out. Add back in later when you have it ready to go, but no benefit in committing as-is